app/acceptance-testing-guide.css

```
@@ -0,0 +1,1112 @@
+                text-align: center;
+                color: var(--ink4);
+                font-size: 12px;
+                font-family: var(--font-mono);
+            }
+            footer a {
+                color: var(--terra);
+                text-decoration: none;
+            }
+
+            /* ── ANIMATIONS ── */
+            @keyframes fadeUp {
```

Stylelint (17.5.0)

[error] Expected keyframe name "fadeUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)